Dockerfile.py3

```diff
@@ -0,0 +1,50 @@
+FROM python:3.6-slim-jessie
+
+MAINTAINER bhearsum@mozilla.com
+
+# Some versions of the python:3.6 Docker image remove libpcre3, which uwsgi needs for routing support to be enabled.
+# Node and npm are to build the frontend. nodejs-legacy is needed by this version of npm. These will get removed after building.
+# libmysqlclient-dev is required to use SQLAlchemy with MySQL, which we do in production.
+# xz-utils is needed to compress production database dumps
+RUN apt-get -q update \
+    && apt-get -q --yes install libpcre3 libpcre3-dev libmysqlclient-dev mysql-client xz-utils \
+    && apt-get clean
+
+WORKDIR /app
+
+# install the requirements into the container first
+# these rarely change and is more cache friendly
+# ... really speeds up building new containers
+COPY requirements.txt /app/
+RUN apt-get install -q --yes gcc && \
+    pip install -r requirements.txt && \
+    apt-get -q --yes remove gcc && \
+    apt-get -q --yes autoremove && \
+    apt-get clean && \
+    rm -rf /root/.cache
+
+# Copying Balrog to /app instead of installing it means that production can run
+# it, and we can bind mount to override it for local development.
+COPY auslib/ /app/auslib/
+COPY ui/ /app/ui/
+COPY uwsgi/ /app/uwsgi/
+COPY scripts/manage-db.py scripts/run-batch-deletes.sh scripts/run.sh scripts/reset-stage-db.sh scripts/get-prod-db-dump.py /app/scripts/
+COPY version.json /app/
+
+WORKDIR /app
+
+RUN cd ui && \
+    apt-get -q --yes install nodejs nodejs-legacy npm && \
```

Comment:
This looks to be a s/2\.7/3.6/g copy of Dockerfile. I tried to see if it will need any package adjustments by comparing  Dockerfile.dev with Dockerfile.py3.dev but they're structured a little differently from this (eg all apt installs in one go). That said, npm, libfontconfig1, and netcat are removed, while gcc is added. So it's possible we'll no longer need npm here; gcc is probably for compiled python deps, seems unlikely to affect nodejs.

Reply:
Don't we still need to build the UI in the py3 image?

Reply:
Absolutely.  I think what's happening is that dev is slightly different to prod - it's using a node docker image which will have npm there already, and not having it in Dockerfile.py3.dev was just cleanup. Meanwhile we do have to install npm in our prod image to make sure we can build the UI, so carry on!